WhisperAPI/WhisperAPI.Tests/Integration/SuggestionsControllerTest.cs

```diff
@@ -146,52 +109,31 @@ public void When_getting_suggestions_with_irrelevant_query_then_returns_empty_li
         }
 
         [Test]
-        [TestCase(System.Net.HttpStatusCode.NotFound)]
-        [TestCase(System.Net.HttpStatusCode.BadRequest)]
-        [TestCase(System.Net.HttpStatusCode.InternalServerError)]
```

Comment:
Why?

Reply:
I just remove the prefix System.Net because it was useless to write the complete path

Reply:
lol of course damn!